data/channels.csv

```diff
@@ -31653,7 +31659,6 @@ VTV.mv,VTV,,,,MV,,,c/MV,div,,FALSE,2008-09-01,,,https://www.vnews.mv/,https://i.
 VTV.ro,VTV,,,,RO,,,c/RO,ron,,FALSE,,,,https://vtv.ro/,https://i.imgur.com/nREmG6e.png
 VTV.rs,VTV,ВТВ,,Srbija danas doo,RS,,Belgrade,c/RS,srp,,FALSE,,,,http://www.novosadska.tv/,https://i.imgur.com/7jCHU4c.png
 VTV.si,VTV,,,,SI,,,c/SI,slv,,FALSE,,,,,https://i.imgur.com/1enjDJ6.png
-VTV.tr,VTV,,,,TR,,,c/TR,tur,,FALSE,,,,,https://i.imgur.com/o26IDy4.png

```

Comment:
Unexplained removal?
https://www.vtv.net.tr/

Reply:
Verify that it belongs to 'Kanal V' by looking at its image.

I mentioned in the PR statement

Reply:
When I have time I will add it again and update the website and picture information.